### ISSUE TITLE
spread syntax did not consider `delegateEvents` option from `babel-plugin-jsx-dom-expressions`

### ISSUE THREAD

**Issue by zhaoyao91** [ryansolid/dom-expressions#48]
see: https://github.com/ryansolid/dom-expressions/blob/fd82f17f6174ff2d69a170fd46b01f5329d9cdcd/packages/dom-expressions/src/runtime.js#L134

I am trying to disable event delegation by setting babel preset as 

```
    [
      "babel-preset-solid",
      {
        "delegateEvents": false
      }
    ]

```

It works for most case, but I found when there is props-spreading syntax in the component, the global delegated handler occurs again. By researching the codebase, I found in the [babel-preset](https://github.com/ryansolid/dom-expressions/blob/fd82f17f6174ff2d69a170fd46b01f5329d9cdcd/packages/babel-plugin-jsx-dom-expressions/src/dom/element.js#L325) it indeed considered the deleteEvents option, but there is no peer consideration in the spread situation.

**Comment by ryansolid** [comment]
I'm removing this behavior on custom events which I'm gathering the issue is. It is hard to actually switch this on and off between the runtime and the compiler. So hopefully the situation will be satisfactory in the next release.

**Comment by ryansolid** [comment]
Alright released in 0.25.0. I will be updating the dependent libraries over the next couple days.
